### ISSUE TITLE
Update Alert Boxes for Accessibility and Alignment with VADS

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#8357]
We completed an audit of the alert boxes found in the mobile app in April of 2024. 

The results can be found [in this Google sheet](https://docs.google.com/spreadsheets/d/1ed9_E3h7FoF8rmQoJV12bcy_ShkKsHf1PrGLrCDGryw/edit?usp=sharing).

**Comment by Sparowhawk** [comment]
@brea11y I believe @narin and @TimRoe are working on the alert box component. This seems like something that should be considered along with several of your other tickets

**Comment by kellylein** [comment]
Per discussion on 4/17/24, this work is all for the Flagship app.  Work to update the Alert component is covered in https://github.com/department-of-veterans-affairs/va-mobile-library/issues/61
